### DISCUSSION TITLE
Plans for VisiData 3.0

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#2029] [General]
Hi, 
I was wondering if there are any plans for releasing a  VisiData 3.0? 

- There have already been some fantastic changes with the guide sheets. 
- There was some discussion of a [command palette](https://github.com/saulpw/visidata/discussions/1906) 
- From the same thread mention of possible hierarchical display of data
-  Also, there is now some basic form support.

Wondering if there is any more time available for a big release?  The VisiData birthday is coming up soon on October 29th.

**Comment by saulpw** [comment]
Hey @frosencrantz!  Yes, I'm aiming to release VisiData 3.0 by the end of the year.  Before then, I'd like to implement, fix, or Kondo most open Github Issues.  @zormit is working on #247 for the "command palette" (Spacebar then start typing and get a fuzzy match of commands).  I'm also hoping to improve the fancy chooser for jointypes and aggregators.

If we're feeling amibitious, I'd like to put together a plan for creating/improving the guide sheets.  I think we have all the pieces now for a builtin tutorial that guides a brand new user through some of the basic commands in a fun way.  If you or anyone else is interested, I'd love the help!
